README.md

```diff
@@ -48,17 +48,16 @@ For further details, please see the complete [security release process](SECURITY
 Please use the following to reach members of the community:
 
 - Slack: Join our [slack channel](https://slack.rook.io)
-- Forums: [rook-dev](https://groups.google.com/forum/#!forum/rook-dev)
+- Github: Start a [discussion](https://github.com/rook/rook/discussions) or open an [issue](https://github.com/rook/rook/issues)
 - Twitter: [@rook_io](https://twitter.com/rook_io)
-- Email (general topics): [cncf-rook-info@lists.cncf.io](mailto:cncf-rook-info@lists.cncf.io)
-- Email (security topics): [cncf-rook-security@lists.cncf.io](mailto:cncf-rook-security@lists.cncf.io)
+- Security topics: [cncf-rook-security@lists.cncf.io](mailto:cncf-rook-security@lists.cncf.io)
```

Comment:
what do you think about also linking to the [security disclosure page](https://github.com/rook/rook/blob/master/SECURITY.md) here too (even though it's mentioned above in its own section), to reinforce the policies and expectations around this security mailing list?

Reply:
How about instead of a `mailto` link, I just link to the section above. So they could still copy the email address if they want to, but if they click on it, they jump to more details.